KP4-Topic3/src/com/aguirre/tests/RomanNumberConverterTest.java

```diff
@@ -0,0 +1,127 @@
+package com.aguirre.tests;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import com.aguirre.classes.RomanNumberConverter;
+
+public class RomanNumberConverterTest {
+
+	@Test
+	public void oneLetter() {

```

Comment:
From my point of view there is no problem with the length of test method, but as I said "from my point of view"